src/cmd/manage_nsfs.js

```diff
@@ -663,6 +664,19 @@ async function get_config_data(config_file_path, show_secrets) {
 ///     VALIDATIONS     ///
 ///////////////////////////
 
+async function is_bucket_empty(fs_context, full_bucket_config_path) {
+    let data;
+    try {
+        data = await get_config_data(full_bucket_config_path);
+    } catch (err){
+        throw_cli_error(ManageCLIError.NoSuchBucket,  full_bucket_config_path);
+    }
+    const entries = await nb_native().fs.readdir(fs_context, data.path);
```

Comment:
we can add a TODO here, once we can read only a single instance in the directory, and not iterate over the whole directory we should use it.